recipes/tvm/cmake_patch.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 494afbdf..9b3413e6 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -51,9 +51,17 @@ tvm_option(USE_ANTLR "Build with ANTLR for Relay parsing" OFF)
+ 
+ # include directories
+ include_directories("include")
+-include_directories("3rdparty/dlpack/include")
+-include_directories("3rdparty/dmlc-core/include")
+-include_directories("3rdparty/rang/include")
++include_directories("${CMAKE_SOURCE_DIR}/include")
++include_directories("${CMAKE_SOURCE_DIR}/include/tvm")
++include_directories("${CMAKE_PREFIX_PATH}/include")
++include_directories("${CMAKE_PREFIX_PATH}/include/dlpack")
++include_directories("${CMAKE_PREFIX_PATH}/include/dmlc")
++include_directories("${CMAKE_PREFIX_PATH}/include/ir")
++include_directories("${CMAKE_PREFIX_PATH}/include/tvm")
++include_directories("${CMAKE_PREFIX_PATH}/include/tvm/node")
++include_directories("${CMAKE_PREFIX_PATH}/include/arithmetic")
++include_directories("${CMAKE_PREFIX_PATH}/include/base")
++include_directories("${CMAKE_PREFIX_PATH}/include/compiler-rt")
+ include_directories("3rdparty/compiler-rt")
```

Comment:
`compiler-rt` is still vendored?

Reply:
It needs this header https://github.com/dmlc/tvm/blob/v0.5/3rdparty/compiler-rt/builtin_fp16.h
but it seems that it is not being packaged by ``compiler-rt``, should I raise it on the feedstock?